music-quiz/src/models/Model.js

```diff
@@ -153,6 +163,10 @@ class Model {
       username: name,
     };
 
+    if (playlist) {
+      newPending["playlist"] = playlist.id;
```

Comment:
I think this is the problem. Try 
`newPending["playlist"] = playlist`
instead. This worked for me.